finetune/mtl.py

```diff
@@ -58,6 +58,7 @@ def get_train_input_fns(self, Xs, Y=None, batch_size=None, val_size=None):
             input_func_normalised, val_func_normalised = get_input_fns(task_id, input_func, val_func)
             input_funcs.append(input_func_normalised)
             val_funcs[task_name] = val_func_normalised
+            val_funcs[task_name + "_train"] = lambda: input_func_normalised().take(val_sizes[task_name])
```

Comment:
This is somehow broken and actually evals for the combined dataset. Not sure how or why. Needs attention

Reply:
Scratch that, doesn't eval for the combined dataset, evals just for the final task, due to input_func_normalised changing. Must define as closure.

Reply:
Fixed